traffic_ops/traffic_ops_golang/config/config.go

```diff
@@ -226,3 +250,22 @@ func ParseConfig(cfg Config) (Config, error) {
 
 	return cfg, nil
 }
+
+func GetLDAPConfig(LDAPConfPath string) (bool, *ConfigLDAP, error) {
+	LDAPConfBytes, err := ioutil.ReadFile(LDAPConfPath)
+	if err != nil {
+
+		return false, nil, fmt.Errorf("reading LDAP conf '%v': %v", LDAPConfPath, err)
+	}
+	LDAPconf, err := getLDAPConf(string(LDAPConfBytes))
+	if err != nil {
+		return false, LDAPconf, fmt.Errorf("parsing LDAP conf '%v': %v", LDAPConfBytes, err)
+	}
+	return true, LDAPconf, nil
+}
+
+func getLDAPConf(s string) (*ConfigLDAP, error) {
+	ldapConf := ConfigLDAP{}
```

Comment:
The LDAP timeout should have a reasonable default. It's defaulting to 0 now, and my LDAP (which is probably misconfigured) is hanging indefinitely. I'd suggest something like 10 seconds, if it isn't set in the config.

Should be accomplished by changing this to
```
const DefaultLDAPTimeout = time.Duration(60 * time.Second)
ldapConf := ConfigLDAP{LDAPTimeoutSecs: DefaultLDAPTimeout / time.Second}
```